source/less/blocks/all-goods.less

```
@@ -0,0 +1,46 @@
+.all-goods{
+  margin-top: 17px;
+  padding: 0 20px 25px 20px;
+  border-bottom: 1px solid @basic_grey-add;
+}
+.all-goods__watch{
+  position: relative;
+  display: block;
+  .size(30px, 30px);
+  margin: 0 auto;
+}
+.plus{
```

```suggestion
.all-goods__plus{
```
Класс `plus` указывает на то, что это БЭМ блок, но это не самостоятельная сущность, а значит нужен класс элемента.